### PR TITLE
Adding Tprime to TZ cards for CASE analysis

### PR DESCRIPTION
**Pull request by wpmccormack**
This is the 3+2 prong dijet topology for the CASE analysis.  Scan over T' and Z masses.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model STP_UFO_freeWidth
+define bb = b b~
+define tt = t t~
+generate p p > t23 > j bb tt z / h
+add process p p > t23~ > j bb tt z / h
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat:9]: Hi, I am not very aware of this model.
t23 goes to t z? And what becomes j b?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/customizecards.dat [hunk 2]
```diff
@@ -0,0 +1,7 @@
+set param_card bsm 1 0.100000e+00
+set param_card bsm 2 0.000000e+00
+set param_card bsm 3 0.100000e+00
+set param_card bsm 4 0.000000e+00
+set param_card bsm 5 0.100000e+00
+set param_card bsm 6 0.000000e+00
+set param_card DECAY 8000001 1.000000e-03
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/customizecards.dat:7]: You will have the same 1MeV width for all the different tprime masses?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat (hunk 1 above)
Comment (wpmccormack) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat:9]: Hi Sihyun,
This is basically just an adaptation of these cards: https://github.com/wpmccormack/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ/tZt_NW/tZ_t_LH_M700.  When I look at the lhe file for this process, I see e.g.:
<event>
13      2 +1.8079137e-02 5.88367800e+01 7.81860800e-03 1.22107000e-01
        1 -1    0    0  501    0 +0.0000000000e+00 +0.0000000000e+00 +2.1080922399e+03 2.1080922399e+03 5.0400000000e-03 0.0000e+00 -1.0000e+00
       21 -1    0    0  502  503 +0.0000000000e+00 +0.0000000000e+00 -1.0677618059e+02 1.0677618059e+02 0.0000000000e+00 0.0000e+00 1.0000e+00
 -8000001  2    1    2    0  503 +9.3935538870e+00 +5.3269099979e+01 +1.3275558431e+03 1.5017765418e+03 7.0000173511e+02 0.0000e+00 0.0000e+00
        2  1    1    2  501    0 -7.9086053433e+00 -5.2719893388e+01 +6.9207485872e+02 6.9412501991e+02 2.5500000000e-03 0.0000e+00 -1.0000e+00
        5  1    1    2  502    0 -1.4849485437e+00 -5.4920659035e-01 -1.8314642536e+01 1.8966858781e+01 4.6700000000e+00 0.0000e+00 1.0000e+00
       -6  2    3    3    0  503 +9.0055722657e+01 -2.3477595110e+02 +1.0132552686e+03 1.0578834954e+03 1.7088437638e+02 0.0000e+00 0.0000e+00
       -5  1    6    6    0  503 +9.0689383231e+01 -5.1817698858e+01 +3.3640708190e+02 3.5227996224e+02 4.6700000000e+00 0.0000e+00 1.0000e+00
      -24  2    6    6    0    0 -6.3366057415e-01 -1.8295825225e+02 +6.7684818669e+02 7.0560353315e+02 7.9238592675e+01 0.0000e+00 0.0000e+00
        3  1    8    8  504    0 +1.2190152472e+01 -1.1048576041e+02 +2.6723564169e+02 2.8943151428e+02 1.0100000000e-01 0.0000e+00 -1.0000e+00
       -2  1    8    8    0  504 -1.2823813046e+01 -7.2472491839e+01 +4.0961254500e+02 4.1617201886e+02 2.5500000000e-03 0.0000e+00 1.0000e+00
       23  2    3    3    0    0 -8.0662168770e+01 +2.8804505108e+02 +3.1430057454e+02 4.4389304641e+02 9.3700846149e+01 0.0000e+00 0.0000e+00
       12  1   11   11    0    0 -1.4615239670e+01 +1.5708851577e+02 +1.0971979994e+02 1.9216878392e+02 0.0000000000e+00 0.0000e+00 -1.0000e+00
      -12  1   11   11    0    0 -6.6046929100e+01 +1.3095653531e+02 +2.0458077460e+02 2.5172426248e+02 0.0000000000e+00 0.0000e+00 1.0000e+00

Basically, there seems to be an ISR jet, then Tprime + B production.  The Tprime then decays to top + Z.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/customizecards.dat (hunk 2 above)
Comment (wpmccormack) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/customizecards.dat:7]: Hi Sihyun, we just want a narrow decay, so I think this is ok for simplicity's sake

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat (hunk 1 above)
Comment (agrohsje) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat:5]: note that with the above syntax lorentz and gauge variance tests fail in mg5. did you ever compare the kinematics etc to p p > t23 j bb , t23 > tt Z ?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/run_card.dat [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  5  = ptj       ! minimum pt for the jets
+  0  = ptb       ! minimum pt for the b
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/run_card.dat:109]: is it on purpose to cut differently on j and b. both just associate the t23.

**Comment by agrohsje** [comment]
I raised a comment here: 
https://github.com/cms-sw/genproductions/pull/2995
Can you comment: what was the problem with generate p p > t23 j bb , t23 > tt Z  ? 0 cross section as in the other PR?

**Comment by wpmccormack** [comment]
> I raised a comment here: #2995 Can you comment: what was the problem with generate p p > t23 j bb , t23 > tt Z ? 0 cross section as in the other PR?

Yes, the issue in both cases was 0 cross section

**Comment by agrohsje** [comment]
Thanks. So what about having in one PR Z > j j inclusind b b~ in the other not.

**Comment by wpmccormack** [comment]
Hi @agrohsje , I added b bbar to the Z decays in this PR.  Primarily, we're just concerned about the pronginess of the Z decay, but you're right that we should be consistent.  Thanks!

**Comment by wpmccormack** [comment]
Hi @agrohsje , another quick status ping here, similar to PR 2994 and 2996.  Thanks!

**Comment by agrohsje** [comment]
Hi @sihyunjeon : are you ok with merging. I am ok with the PR. Thanks.

**Comment by wpmccormack** [comment]
Hi Both, I set the decay width of Tprime to 3% of the mass to synchronize with the other PRs (2995 and 2996)
